internal/bminventory/inventory_test.go

```diff
@@ -2077,6 +2077,31 @@ var _ = Describe("cluster", func() {
 			bm.clusterApi = cluster.NewManager(cluster.Config{}, common.GetTestLog().WithField("pkg", "cluster-monitor"),
 				db, mockEvents, nil, nil, nil, nil, nil)
 
+			mockEvents.EXPECT().
+				AddEvent(gomock.Any(), gomock.Any(), nil, models.EventSeverityInfo, gomock.Any(), gomock.Any()).
+				Times(2)
+			mockMetric.EXPECT().ClusterRegistered(gomock.Any(), gomock.Any(), gomock.Any()).Times(1)
+			mockVersions.EXPECT().IsOpenshiftVersionSupported(gomock.Any()).Return(true).Times(1)
+			noneHaMode := models.ClusterHighAvailabilityModeNone
+			MinimalOpenShiftVersionForNoneHA := "4.8"
+			reply := bm.RegisterCluster(ctx, installer.RegisterClusterParams{
+				NewClusterParams: &models.ClusterCreateParams{
+					Name:                 swag.String("some-cluster-name"),
+					OpenshiftVersion:     swag.String(MinimalOpenShiftVersionForNoneHA),
+					PullSecret:           swag.String("{\"auths\":{\"cloud.openshift.com\":{\"auth\":\"dG9rZW46dGVzdAo=\",\"email\":\"coyote@acme.com\"}}}"),
+					HighAvailabilityMode: &noneHaMode,
+				},
+			})
+			Expect(reflect.TypeOf(reply)).Should(Equal(reflect.TypeOf(installer.NewRegisterClusterCreated())))
+			actual := reply.(*installer.RegisterClusterCreated)
+			Expect(actual.Payload.HighAvailabilityMode).To(Equal(swag.String(noneHaMode)))
+			Expect(actual.Payload.UserManagedNetworking).To(Equal(swag.Bool(true)))
+		})
+
+		It("create non ha cluster fail", func() {
```

Comment:
Can this test be more explicit about using a lower version number?

Reply:
:+1: